### PR TITLE
Fix colour being doubled for Paths

### PR DESCRIPTION
**Pull request by smoogipoo**

### CODE REVIEW

File: osu.Framework/Graphics/Lines/Path.cs [hunk 1]
```diff
@@ -176,18 +176,12 @@ protected Texture Texture
...
-                result.Blending = new BlendingInfo(BlendingMode.None);
-                return result;
-            }
-        }
-
         public DrawColourInfo? FrameBufferDrawColour => base.DrawColourInfo;
 
+        // Children should not receive the true colour to avoid colour doubling when the frame-buffers are rendered to the back-buffer.
+        // Removal of blending allows for correct blending between the wedges of the path.
+        public override DrawColourInfo DrawColourInfo => new DrawColourInfo(Color4.White, new BlendingInfo(BlendingMode.None));
```
Comment (Tom94) [inline osu.Framework/Graphics/Lines/Path.cs]: I am not sure this is correct. How do paths get the colors of their parent, now?

Comment (smoogipoo) [reply osu.Framework/Graphics/Lines/Path.cs]: They're buffered:

https://github.com/ppy/osu-framework/blob/92e3a190a6ac65e0c07ccd19d7af0c562948fc1b/osu.Framework/Graphics/Lines/Path.cs#L189

So they get it when it's applied to the framebuffer itself:

https://github.com/ppy/osu-framework/blob/92e3a190a6ac65e0c07ccd19d7af0c562948fc1b/osu.Framework/Graphics/Lines/Path.cs#L179

Comment (Tom94) [reply osu.Framework/Graphics/Lines/Path.cs]: Aaaah, I see, I didn't realize they are buffered by now. Looks good, although the comment is a little bit misleading as it talks about "children". I don't think paths can actually have children, can they?

Comment (smoogipoo) [reply osu.Framework/Graphics/Lines/Path.cs]: Fair enough, this was a copy from BufferedContainer, will adjust.

I really don't like this stuff to be honest. I want to handle it in a better way at a DrawNode-only level, but I haven't been able to think up a good structure for it yet.

**Pull request merged**